.cloudbuild/library_generation/cloudbuild-library-generation-push.yaml

```diff
@@ -14,7 +14,7 @@
 
 timeout: 7200s # 2 hours
 substitutions:
-  _IMAGE_NAME: "gcr.io/cloud-devrel-public-resources/java-library-generation"
+  _IMAGE_NAME: "us-docker.pkg.dev/java-hermetic-build-prod/java-hermetic-build-prod-images"
```

Comment:
I think the project name `java-hermetic-build-prod` is a generated name from `java-hermetic-build`? I don't think we need to repeat `java-hermetic-build-prod` again in the image name. Can we choose a different image name? `library-generation` actually sounds fine to me, but you may have a better idea.

Reply:
I'll take a look about how to change the image name.

Reply:
Repository name updated in cl/679753674.